test/test_euclvec.jl

```diff
@@ -20,29 +19,39 @@ B1p = ACE.Utils.RnYlm_1pbasis(; maxdeg=maxdeg, D = D)
 
 # generate a configuration
 nX = 10
-X0 = rand(EuclideanVectorState, B1p.bases[1])
 Xs = rand(EuclideanVectorState, B1p.bases[1], nX)
+cfg = ACEConfig(Xs)
 
```

Comment:
These are my changes : I've generalized and abstracted the concept of a "configuration";  I hope this will help us in the future.